apps/preview/nuxt/pages/showcases/MegaMenu/MegaMenuNavigation.vue

```diff
@@ -0,0 +1,604 @@
+            variant="tertiary"
+            square
+            aria-label="Close menu"
+            class="block md:hidden mr-5 bg-transparent hover:bg-primary-800 hover:text-white active:bg-primary-900 active:text-white"
+          >
+            <SfIconMenu class="text-white" />
+          </SfButton>
+          <a
+            href="/"
+            aria-label="SF Homepage"
+            class="flex items-center mr-2 text-white md:mr-10 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
+          >
+            <picture>
+              <source :srcset="brandLogo" media="(min-width: 1024px)" />
+              <img :src="brandLogoSign" alt="Sf Logo" class="w-8 h-8 lg:w-[12.5rem] lg:h-[1.75rem]" />
+            </picture>
+          </a>
+        </div>
+        <div class="flex flex-nowrap">
+          <SfButton
+            v-for="actionItem in actionItems"
+            :key="actionItem.ariaLabel"
+            :aria-label="actionItem.ariaLabel"
+            class="ml-2 text-white bg-transparent hover:bg-primary-800 hover:text-white active:bg-primary-900 active:text-white"
+            variant="tertiary"
+            square
+          >
+            <template #prefix>
+              <Component :is="actionItem.icon" />
+            </template>
+            <p v-if="actionItem.role === 'login'" class="hidden md:inline-flex">{{ actionItem.label }}</p>
+          </SfButton>
+        </div>
+      </div>
+
+      <nav ref="floatingRef">
+        <ul
+          class="hidden md:flex px-6 py-2 bg-white border-b border-b-neutral-200 border-b-solid"
+          @blur="
+            (event) => {
+              if (!(event.currentTarget as Element).contains((event.relatedTarget as Element))) {
+                close();
+              }
+            }
+          "
+        >
+          <li v-for="node in content.children" key="node.key">
+            <SfButton variant="tertiary" class="mr-2" @mouseenter="openMenu([node.key])" @click="openMenu([node.key])">
+              <span>{{ node.value.label }}</span>
+              <SfIconChevronRight class="rotate-90" />
+            </SfButton>
+
+            <div
+              v-if="isOpen && activeNode.length === 1 && activeNode[0] === node.key"
+              :key="activeMenu.key"
+              :style="style"
+              class="hidden md:grid gap-x-6 grid-cols-4 bg-white shadow-lg p-6 left-0 right-0"
+              @mouseleave="close()"
+            >
+              <template v-for="node in activeMenu.children" :key="node.key">
+                <div v-if="node.isLeaf" class="mb-2">
+                  <SfListItem tag="a" size="sm" role="none" :href="node.value.link" class="typography-text-sm">
+                    {{ node.value.label }}
+                  </SfListItem>
+                </div>
+                <div v-if="node.isLeaf" class="col-start-2 col-end-5"></div>
+                <div v-else>
+                  <h2
+                    role="presentation"
```

Comment:
In some cases you use `role="none"` and in some `role="presentation"`. Let's unify this to "presentation" everywhere - what do you think?

Reply:
Since `role="presentation"` removes semantics from the heading element, I decided to change it to `<p>` and remove `role` attribute.